test/e2e/run_test.go

```diff
@@ -48,6 +48,26 @@ var _ = Describe("Podman run", func() {
 		Expect(session).Should(Exit(0))
 	})
 
+	It("podman run two containers choose different names", func() {
+		session1 := podmanTest.Podman([]string{"run", "-d", ALPINE, "sleep 1m"})
+		session2 := podmanTest.Podman([]string{"run", "-d", ALPINE, "sleep 1m"})
```

Comment:
This can't possibly work, and it's going to fail CI with something like "no such executable, 'sleep 1m'". Needs to be `"sleep", "1m"` or something like that, separate tokens.